test/integration/services/data/test-get-claims-list-and-count.js

```diff
@@ -37,7 +38,17 @@ describe('services/data/get-claim-list-and-count', function () {
     })
 
     it('should return no data for status with no records', function () {
-      return getClaimsListAndCount('TESTING_NONE', false, 0, 10)
+      return getClaimsListAndCount('TESTING_NONE', false, 0, 10, 'TestUser@test.com')
+        .then(function (result) {
+          expect(result.total.Count).to.equal(0)
+        })
+        .catch(function (error) {
+          throw error
+        })
+    })
+
+    it('should return no data for records assigned to another user', function () {
```

Comment:
add two new test cases, one for claim AssignedTo is null and one for AssignmentExpiry is past